### DISCUSSION TITLE
Wrong location for context menus in virtual monitors created by xrandr in an ultrawide monitor

### DISCUSSION THREAD

**Discussion by nadrad** [i3/i3#5315] [Q&A] [answered]
I use xrandr to split my ultrawide monitor into 3 virtual monitors:

```bash
xrandr \
   --delmonitor H3 \
   --delmonitor H2 \
   --delmonitor H1 \
   --setmonitor H1 1245/290x1440/336+0+0     HDMI-0 \
   --setmonitor H2 1670/389x1440/336+1245+0  none \
   --setmonitor H3 2205/514x1440/336+2915+0  none
```

It works relatively fine, except that the context menus and similar frames on the second and third virtual monitor open on the right border of the first virtual monitor instead of their regular position relative to their window.

Is there something I should do differently to avoid this issue?

**Comment by vincentbernat** [comment, marked as answer]
This is a bug in GTK but GTK maintainers are unwilling to fix it despite several solutions were proposed. See https://gitlab.gnome.org/GNOME/gtk/-/issues/2013.
